samples/samples-csharp/OutputBindingSamples/AddProductsArray.cs

```diff
@@ -1,39 +1,25 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+using System.Collections.Generic;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.OutputBindingSamples
 {
     public static class AddProductsArray
     {
         [FunctionName("AddProductsArray")]
         public static IActionResult Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "addproducts-array")]
-            HttpRequest req,
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "addproducts-array")]
+            [FromBody] List<Product> products,
             [Sql("dbo.Products", ConnectionStringSetting = "SqlConnectionString")] out Product[] output)
         {
             // Suppose that the ProductID column is the primary key in the Products table, and the
             // table already contains a row with ProductID = 1. In that case, the row will be updated
             // instead of inserted to have values Name = "Cup" and Cost = 2.
```

Comment:
Let's update this comment as well so it's more accurate. (and add this comment to the JS function version as well)
```
Upsert the products, which will insert them into the Products table if the primary key (ProductId) for that item doesn't exist. If it does then update it to have the new name and cost
```